lib/compute/agent-nodes.ts

```diff
@@ -184,19 +182,6 @@ export class AgentNodes {
       initScript: 'echo',
       remoteFs: '/var/jenkins',
     };
-    this.WINDOWS2019_X64 = {
```

Comment:
We no longer use this @peterzhuamazon ?

Reply:
Nope, we have since changed to windows docker in 2.10.0.